### DISCUSSION TITLE
Access to data (uploaded to the inbox) using EGA website

### DISCUSSION THREAD

**Discussion by josefhandl** [neicnordic/sensitive-data-archive#881] [Q&A]
Hello, I would like to ask for clarification about accessing uploaded data.

I have installed the SDA, logged in to the `auth` service, downloaded the `s3cmd.conf` file, uploaded data to the `inbox`, but I don't see anything on the EGA website. I used a LS AAI for the authentication to the `auth`, and linked (I hope) LS AAI account with EGA account. However the `Files in inbox` page is empty.

I have verified that the data is uploaded to the s3 server. There are 3 buckets: `archive`, `backuparchive` and `inbox`. And the `inbox` contains directory with my LS ID in the format `<ls_id>_elixir-europe.org` and the uploaded data in it.

Do I need to do any other steps to make this work? Maybe I don't understand the mechanism of how does the data access is supposed to work.

**Comment by jbygdell** [comment]
In the FEGA case you also have to check the MQ.
* Log in to the management interface and verify that the shovels are working correctly, (admin tab -> shovel status) all 5 should be running.
* Next verify that the federation link is working, (admin tab -> federation status)
* The local `inbox` and `catch-all.dead` queues should be empty.

If this is a test setup you can also log in to your FEGA MQ and check that no messages can be found in the `catch_all.dead` or `error` queues.

**Reply by josefhandl** [reply]
Unfortunately both the `shovel status` and the `federation status` tabs are empty. I have also look at the queues on the `rabbitmq` website, and the `catch_all.dead` contains one message. The other queues are empty.
